Xamarin.Forms.Core/PromptArguments.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Xamarin.Forms.Internals
+{
+	public class PromptArguments
+	{
+		public PromptArguments(string title, string message, string accept, string cancel, string placeholder = null, int? maxLength = null, Keyboard keyboard = default(Keyboard))
+		{
+			if (string.IsNullOrWhiteSpace(accept) || string.IsNullOrWhiteSpace(cancel))
```

Comment:
users shouldn't call this directly, so move the arg validation to the caller

Reply:
If we move it to the caller, there is no need to do validation since both of these are defaulted to "OK" and "Cancel" respectively in the caller. So, I guess I'll remove the validation.